testing/README.md

```diff
@@ -0,0 +1,13 @@
+# Manually Testing Tentacle
+
+## Compatability Testing
+
+The following Vagrant and Docker scripts are available to provision Tentacles and assist with manually setting up different versions of Tentacle for compatibility testing.
```

Comment:
We have Azure scripts now. Maybe we should update this readme to reflect that as well?